cmd/massastation/main.go

```diff
@@ -63,6 +71,10 @@ func main() {
 		logger.Fatalf("Error with you current system configuration: %s", err.Error())
 	}
 
+	if startFlags.Repair {
+		os.Exit(0)
```

Comment:
What about adding a message informing the enduser that  the execution was finished.